src/types.d.ts

```diff
@@ -0,0 +1,32 @@
+export type Column = {
```

Comment:
Add comment:

Most of the JS classes/functions have type information as JSDoc (included in the corresponding comments). For some types this was not possible (e.g. because we didn't have actual classes), hence we define them here. They are "included" by the class Timeline.

Reply:
Rezolvat.